packages/front/svelte/helpers/providerValueHelper.test.js

```diff
@@ -1,13 +1,33 @@
 import * as ProviderValueHelper from "./providerValueHelper";
 
-describe("providerValueHelper", () => {
-    const PROVIDER_VALUE = {
-        value: "VALUE",
-        provider: "PROVIDER",
-        last_update: new Date(),
+function buildProviderValue(value, providerName = "PROVIDER A", date = new Date()) {
+    return {
+        value: value,
+        provider: providerName,
+        last_update: date,
     };
+}
+
+describe("providerValueHelper", () => {
+    const PROVIDER_VALUE = buildProviderValue("VALUE");
+    const today = new Date();
+    const yesterday = new Date(today);
+    yesterday.setDate(yesterday.getDate() - 1);
 
-    // TODO: flatenProviderValue
+    describe("flatenProviderValue", () => {
+        describe("when value is a ProviderValues", () => {
```

Comment:
l'intitulé du describe n'est pas explicite

Reply:
tu préfères "when param is a ProviderValues" ? Les majuscules sont là pour identifier le type d"objet